src/ZuriAC.php

```diff
@@ -123,8 +123,13 @@ public function onLoad() : void {
 		self::$instance = $this;
 		ConfigManager::checkConfig();
 
+                if (!version_compare(PHP_VERSION, '8.3.0', '>=')) {
+                    $this->getLogger()->error("⚠️ You're running PHP " . PHP_VERSION . ", which is older than 8.3.0. Please upgrade your PHP Installion to 8.3.0 or later!");
```

Comment:
8.3.0 is used 2 times and I think it should be put in a constant instead of hardcode